src/auth/presentation/guards/optional-jwt-auth.guard.ts

```
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/* eslint-disable @typescript-eslint/no-unsafe-return */
+import { Injectable, ExecutionContext } from '@nestjs/common';
```

ESLint

[error] Replace `Injectable,·ExecutionContext` with `ExecutionContext,·Injectable`

(prettier/prettier)